cmd/desync/extract.go

```diff
@@ -164,14 +168,33 @@ func writeInplace(ctx context.Context, name string, idx desync.Index, s desync.S
 	return desync.AssembleFile(ctx, name, idx, s, seeds, assembleOpt)
 }
 
-func readSeeds(dstFile string, locations []string, opts cmdStoreOptions) ([]desync.Seed, error) {
+func readSeeds(dstFile string, seedsInfo []string, opts cmdStoreOptions) ([]desync.Seed, error) {
 	var seeds []desync.Seed
-	for _, srcIndexFile := range locations {
+	for _, seedInfo := range seedsInfo {
+		var (
+			srcIndexFile string
+			srcFile      string
+		)
+
+		if strings.HasSuffix(seedInfo, ".caibx") {
+			srcIndexFile = seedInfo
+			srcFile = strings.TrimSuffix(srcIndexFile, ".caibx")
+		} else {
+			seedArray := strings.SplitN(seedInfo, ":", 3)
```

Comment:
You don't really need to use SplitN here, a simple Split() would do as well which then removes the magic 3 and make it more clear

Reply:
Good point. I changed it to `Split()`.